### DISCUSSION TITLE
Click on the button link and jump with an error.

### DISCUSSION THREAD

**Discussion by tttbear** [JulianPrieber/littlelink-custom#339] [Feedback]
> docker

I am having some problems with it and would like to ask for your help. Thank you very much for sharing LittleLink.  

1. When "Blocks" selects "Custom Link", you can save successfully without filling in the URL. This causes an error when clicking the button. Suggestion: When the url is empty, clicking the button will not respond, or the button text will be copied automatically.

2. When "Phone" is selected for the "predefined site" item, you will be prompted to enter a URL. It should be that when the user clicks the button, the number will be automatically copied and the prompt will be successful. No URL is required.

3. I choose not to verify email when I install littlelink in docker, but when I log into the backend panel, it is turned on by default in the advanced settings and needs to be turned off again.  
After completing the above, I tried to register a new account. When entering the email and password and clicking submit it reports an error (I forgot to take a screenshot of the exact code). I plan to reinstall it again, I will give feedback on the specific problem next time.   

Thanks again and may you be happy every day!  

![01](https://user-images.githubusercontent.com/43196479/222172951-3f393c23-b6e8-4b27-8d38-a5c09ea2eeed.png)

**Comment by JulianPrieber** [comment]
Hey @tttbear thanks for the feedback!

> When "Blocks" selects "Custom Link", you can save successfully without filling in the URL. This causes an error when clicking the button. Suggestion: When the url is empty, clicking the button will not respond, or the button text will be copied automatically.

Should be fixed in the latest patch `3.4.1`

<br>

> When "Phone" is selected for the "predefined site" item, you will be prompted to enter a URL. It should be that when the user clicks the button, the number will be automatically copied and the prompt will be successful. No URL is required.

The phone button used to allow you to enter just a phone number, then when clicked on the links page would open a dialog box that allows you to call or save that number.
This was changed with the 3.0 update, which updated the system we use to add links with a user submission. Here all inputs have to be links.

You can still use this functionality as you can actually save phone numbers as URL. 
Similar to the email button, this is a valid URL: [tel:1234567890](tel:123)
For the mail, it's this: [mailto:info@llc.ovh](mailto:info@llc.ovh)

We plan on adding dedicated email and telephone blocks that do what the old button did. We experienced some difficulties including this, so it's not certain when this will be added.

<br>

> I choose not to verify email when I install littlelink in docker, but when I log into the backend panel, it is turned on by default in the advanced settings and needs to be turned off again.
> After completing the above, I tried to register a new account. When entering the email and password and clicking submit it reports an error (I forgot to take a screenshot of the exact code). I plan to reinstall it again, I will give feedback on the specific problem next time.

For this one, you need to help me understand a couple of things. 

To confirm this, I set up a new instance with docker. I selected to disable email verification and after completing the installation the setting was correctly disabled in the config.
I'm not sure what you're referring to with "advanced settings". I'm assuming you mean the advanced section of the config page?
The setting "Enable email verification" cannot be found in the advanced section. It's the second setting from the top:
![chrome_G4ZlyIb1Gd](https://user-images.githubusercontent.com/60265788/222196543-e936a48e-0c80-4e7a-ab3d-f8a5f0b29f5b.png)

The error you're describing is probably a known one and can mostly be ignored for now.
It was probably this one:
![chrome_4Hdwz7wNSP](https://user-images.githubusercontent.com/60265788/222196815-cfe78769-2933-4807-9fda-2573ea8cad56.png)

We're working on fixing this one. We used to provide a free SMTP server with every instance, hosted by us. The problem is the domain we used to use for emails does now no longer resolve.
We're working on restoring our SMTP server and fixing the issue. We're quite tight on our budget, so this might take a minute until we can figure something out.

<br>

**Happy that you're enjoying our app!**

_If you don't mind, I will be converting this into a discussion after this. If you want to add or clarify anything to this, please continue there 👍._
